batch/src/main/java/org/openmrs/analytics/JdbcFetchUtil.java

```diff
@@ -167,40 +168,37 @@ public Integer fetchMaxId(String tableName) throws SQLException {
 		return rangeMap;
 	}
 	
-	public Map<String, String> createFhirReverseMap(String searchString, String tableFhirMapPath) throws Exception {
+	public Map<String, ArrayList<String>> createFhirReverseMap(String searchString, String tableFhirMapPath)
```

Comment:
nit: Use interface `List` instead of `ArrayList` in the API.
Also I think this public function is complex enough to warrant a javadoc section.